Makefile

```diff
@@ -60,25 +60,38 @@ $(OPERATOR): vendor
 install-ginkgo:
 	go get -u github.com/onsi/ginkgo/ginkgo
 	go get -u github.com/onsi/gomega/...
+	ginkgo version
+
+install-gocov:
+	cd && GO111MODULE=on go get github.com/axw/gocov/gocov@v1.0.0
```

Comment:
Because that's how I understand Go modules based on https://dev.to/maelvls/why-is-go111module-everywhere-and-everything-about-go-modules-24k:

The statements in `install-ginkgo` are installing a tool without a version number. I consider this not ideal. Remember the issue with the kind v0.8 update recently that broke our build. So, I chose to specifically install the current version v1.0.0. If I do just a `go get github.com/axw/gocov/gocov@v1.0.0`, then this will modify the `go.mod` of the project which I do not want. So, I want to jump out of the project scope which is why `cd` is there. In the global scope I am in GOPATH mode. In this mode, I cannot install a specific version without enabled Go modules, therefore `GO111MODULE=on`.

Reply:
OK, thanks for the clarification.

So if we execute the `install-gocov`, will it update the new different  version gocov go file in the vendor folder and show us an uncommit ted change? If no, I am ok for this code.

Reply:
Running `make install-gocov` does not cause uncommitted changes.

Reply:
Thanks for the comment, look good to me!